htlcswitch/switch_test.go

```diff
@@ -3287,7 +3287,7 @@ func TestSwitchHoldForward(t *testing.T) {
 	assertNumCircuits(t, s, 0, 0)
 	assertOutgoingLinkReceive(t, bobChannelLink, false)
 
-	require.NoError(t, switchForwardInterceptor.resolve(&FwdResolution{
+	require.NoError(t, switchForwardInterceptor.Resolve(&FwdResolution{
```

Comment:
I think this fixes: https://github.com/lightningnetwork/lnd/issues/5437 ? Or something similar to it.

Reply:
That commit was pulled in from https://github.com/lightningnetwork/lnd/pull/6353 which is merged now. @bhandras do you think https://github.com/lightningnetwork/lnd/pull/6353 fixed https://github.com/lightningnetwork/lnd/issues/5437 or was that something different?